build.gradle

```diff
@@ -159,16 +159,12 @@ task goPostCommit() {
 
 task pythonPreCommit() {
   dependsOn ":rat"
-  dependsOn ":beam-sdks-python:check"
+  dependsOn ":beam-sdks-python:preCommit"
 }
 
 task pythonPostCommit() {
-  doLast {
-    exec {
-      executable 'sh'
-      args '-c', './sdks/python/run_postcommit.sh'
-    }
-  }
+  dependsOn ":beam-sdks-python:preCommit"
```

Comment:
Redundant, since postCommit already dependsOn preCommit (in sdks/python/build.gradle).

Reply:
Good catch. Thanks!